design/Google Play Store prerequisites.txt

```diff
@@ -0,0 +1,10 @@
+Steps to Sign App
+=================
+The detail of the steps of how to prepare the release build can be found in the following url,
+https://developer.android.com/studio/publish/app-signing
+
+The summary of the steps are,
+1. Generate an upload key and keystore. This is a one time step and the key has to be save in a secure place and passwords too. The same key needs to be used for the future releases.
```

Comment:
Rafi please list the actual commands you actual ran to do this for vector/act_riot_key_store.jks

Reply:
The process is with a GUI embedded in Android Studio. There is no need of running any command for creating the key or creating the build. The process is really smooth and easy.